msal/src/main/java/com/microsoft/identity/client/PublicClientApplication.java

```diff
@@ -969,10 +1165,6 @@ private static void validateAccountModeConfiguration(@NonNull final PublicClient
                     "Warning! B2C applications should use MultipleAccountPublicClientApplication. "
                             + "Use of SingleAccount mode with multiple IEF policies is unsupported."
             );
-
-            if (config.getAuthorities().size() > 1) {
```

Comment:
I don't think we should have removed this. I just looked through the code and I can't find a reason why we would have. Let's put it back.